README.md

```diff
@@ -372,13 +372,12 @@ Due to a critical bug in Agent versions `6.14.0` and `6.14.1` on Windows, these
 been blacklisted (starting with the version `3.3.0` of this role).
 
 **PLEASE NOTE:** ansible will fail on Windows if `datadog_agent_version` is set
-to `6.14.0` or `6.14.1`. Please use `6.14.2` instead. If the Agent version is not
-pinned, `6.14.2` will be installed by default.
+to `6.14.0` or `6.14.1`. Please use `6.14.2` or above instead.
 
 If you are updating from **6.14.0 or 6.14.1 on Windows**, we **strongly** recommend following these steps:
 
 1. Upgrade the present `Datadog.datadog` ansible role to the latest version (`>=3.3.0`)
-2. Set the `datadog_agent_version` to `6.14.2`
+2. Set the `datadog_agent_version` to `6.14.2` or above (by default the role install latest).
```

Comment:
Is using the default role behavior also an option? In that case I'd put something like this to stress the fact that using latest is also fine:
```suggestion
2. Set the `datadog_agent_version` to `6.14.2` or above (or use the default role behavior, which will install the latest Agent 6).
```

Reply:
If not specify the role will install the latest version. I think we should avoid mentioning `6` since `7` is around the corner.